qmetaobject/src/qttypes.rs

```diff
@@ -124,9 +124,17 @@ impl QDate {
     pub fn get_y_m_d(&self) -> (i32, i32, i32) {
         let mut res = (0, 0, 0);
         let (ref mut y, ref mut m, ref mut d) = res;
+
+        // In version prior to Qt 5.7, this method was marked non-const.
```

Comment:
I think it's  fine to keep the 5.6 variant.

Reply:
Hmm. While I would be okay with that, I do wonder whether the Rust signature should be `pub fn get_y_m_d(&mut self)` instead for Qt 5.6, if we'd be really pedantic.

Reply:
I've kept the non-const version.